content/docs/authorization/deployment/_index.md

```diff
@@ -72,7 +72,7 @@ A Storage Administrator can execute the installer or rpm package as a root user
 
 2. In order to configure secure grpc connectivity, an additional subdomain in the format `grpc.DNS_host_name` is also required. All traffic from `grpc.DNS_host_name` needs to be routed to `DNS_host_name` address, this can be configured by adding a new DNS entry for `grpc.DNS_host_name` or providing a temporary path in the `/etc/hosts` file.  
 
-    **NOTE:** The certificate provided in `crtFile` should be valid for both the `DNS_host_name` and the `grpc.DNS_host_name` address.  
+>Note: The certificate provided in `crtFile` should be valid for both the `DNS_host_name` and the `grpc.DNS_host_name` address.  
```

Comment:
Generally, `Note` is written in this form `> __Note__:`

Reply:
fixed